src/config.c

```diff
@@ -228,6 +273,15 @@ int RAI_configParamParse(RedisModuleCtx *ctx, const char *key,
       RedisModule_Log(ctx, "notice", buffer);
       RedisModule_Free(buffer);
     }
+  } else if (strcasecmp((key), "MODEL_CHUNK_SIZE") == 0) {
+    RedisAI_Config_ModelChunkSize(rsval);
+    char *buffer = RedisModule_Alloc(
+        (3 + strlen(REDISAI_INFOMSG_MODEL_CHUNK_SIZE) + strlen((val))) *
+        sizeof(*buffer));
+    sprintf(buffer, "%s: %lld", REDISAI_INFOMSG_MODEL_CHUNK_SIZE,
+            getModelChunkSize());
+    RedisModule_Log(ctx, "notice", buffer);
```

Comment:
`void REDISMODULE_API_FUNC(RedisModule_Log)(RedisModuleCtx *ctx, const char *level, const char *fmt, ...)`
I think you can do
`RedisModule_Log(ctx, "notice", "%s: %lld", REDISAI_INFOMSG_MODEL_CHUNK_SIZE,
            getModelChunkSize());`

Reply:
Done